functional-tests/bdd-tests/Catering_licence_download.feature

```diff
@@ -0,0 +1,11 @@
+﻿Feature: Catering_licence_download
+    As a logged in business user
+    I would like to download a catering licence
+
+Scenario: Download Catering Licence
+    Given the Catering application has been approved
+    And I am logged in to the dashboard as a private corporation
+    And I click on the Licences tab
+    And the licence fee has been paid
+    And I click on the Download Licence link
+    Then the licence is downloaded
```

Comment:
do you have the ability to confirm that certain elements are available on the PDF? If possible I would like to confirm:
* expiry date exists
* address is correct
* terms and conditions exist